app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -212,7 +209,15 @@ private fun ConversationScreenContent(
         messageText = messageText,
         onMessageChanged = onMessageChanged,
         onSendButtonClicked = onSendButtonClicked,
-        onSendAttachment = onSendAttachment,
+        onSendAttachment = {
+            coroutineScope.launch {
+                onSendAttachment(it)
+                // We add some extra delay to allowing some time to insert the message in the database, display it in the screen and be able
+                // to scroll to the bottom of the chat with the new list of messages
+                delay(2000)
```

Comment:
Maybe we could extract this value to a constant or var, either of those, should do it =)